lib/libzpool/uzfs_io.c

```diff
@@ -29,6 +29,7 @@
 
 #if DEBUG
 inject_error_t	inject_error;
+inject_rebuild_error_t inject_rebuild_error;
```

Comment:
I meant this to be part of inject_error_t itself as separate structure like:
```
typedef struct inject_rebuild_error_s {
	uint64_t dw_replica_rebuild_error_io;
} inject_rebuild_error_t;
typedef struct inject_error_s {
 	inject_delay_t delay;
 	inject_rebuild_error_t inject_rebuild_error;
} inject_error_t;
```

Reply:
ok